Tests/Microsoft.AppCenter.Crashes.Test.Windows/CrashesTest.cs

```diff
@@ -201,5 +213,164 @@ public void ProcessPendingErrorsDoesNotCrashOnFileDeletionFailure()
             Mock.Get(ErrorLogHelper.Instance).Verify(instance => instance.InstanceRemoveStoredErrorLogFile(It.IsAny<Guid>()), Times.Never());
             Mock.Get(mockFile).Verify(file => file.Delete(), Times.Once());
         }
+
+        [TestMethod]
+        public void PlatformNotifyUserConfirmationDefault()
+        {
+            ErrorLogHelper.Instance = GenerateErrorLogHelperWithPendingFile();
+
+            // Start Crashes.
+            Crashes.SetEnabledAsync(true).Wait();
+            Crashes.Instance.OnChannelGroupReady(_mockChannelGroup.Object, string.Empty);
+            Crashes.Instance.ProcessPendingErrorsTask.Wait();
+
+            // Verify the value has been set.
+            var alwaysSendValue = AppCenter.Instance.ApplicationSettings.GetValue(Crashes.PrefKeyAlwaysSend, true);
+            Assert.IsFalse(alwaysSendValue);
+        }
+
+        [TestMethod]
+        public void SendCrashReportsOrAwaitUserConfirmationAsyncAlwaysTrue()
+        {
+            ErrorLogHelper.Instance = GenerateErrorLogHelperWithPendingFile();
+
+            // Start Crashes.
+            Crashes.SetEnabledAsync(true).Wait();
+            Crashes.NotifyUserConfirmation(UserConfirmation.AlwaysSend);
```

Comment:
Is it realistic for `NotifyUserConfirmation` to be invoked before `OnChannelGroupReady`? If the value in preferences needs to be set ahead of start time, perhaps place it in the preferences directly?